### ISSUE TITLE
Dealing with angles > 90

### ISSUE THREAD

**Issue by parad0x96** [sbrunner/deskew#519]
It seems to fail with angles > 90

**Comment by sbrunner** [comment]
Nobody can help you with no more information,
I just can say that it's possible that setting the argument `angle_pm_90` to `True` can help.

**Comment by parad0x96** [comment]
> Nobody can help you with no more information, I just can say that it's possible that setting the argument `angle_pm_90` to `True` can help.

You're right sorry for the lack of information, I have an image which is upside down (180 degrees), it seems to output 0 degrees when using
`angle_pm_90` set to `True`
